### ISSUE TITLE
modules/local-log-export does not provide way to set locked attribute

### ISSUE THREAD

**Issue by sbernier-corp** [terraform-google-modules/terraform-google-log-export#170]
### TL;DR

modules/local-log-export does not provide way to set locked attribute value.

### Expected behavior

Give ability to define the locked parameter, otherwhise the locked parameter is set to null by default and so to false.

### Observed behavior

Default value is false.
And when working with locked bucket we get the following changes which is not what we want ...
  # module.local_log_export["access_approval"].module.destination_log_bucket.google_logging_project_bucket_config.bucket will be updated in-place
  ~ resource "google_logging_project_bucket_config" "bucket" {
        id               = "projects/prj-shared-log-core-02ec/locations/europe-west1/buckets/shared-log-core-access-approval"
      - locked           = true -> null
        name             = "projects/prj-shared-log-core-02ec/locations/europe-west1/buckets/shared-log-core-access-approval"
        # (6 unchanged attributes hidden)
    }

### Terraform Configuration

```hcl
module "local_log_export" {
  source           = "../../modules/local-log-export"
  for_each         = var.log-export-config
  log_sink_filter  = each.value.log_sink_filter
  log_destination_name  = each.value.log_destination_name
  project_id       = lookup(var.projects, "prj-shared-log-core","")
  org_id           = var.org_id
  depends_on       = [google_project_iam_member.sa_stack_permissions]
}
```

### Terraform Version

```sh
1.3.9
```

**Comment by dlambert-eworking** [comment]
I'have got the same issue.
Temporary solved with lifecycle/ignore_changes

When do expect to managed locked attribut in the mogbucket module.
Regards

========================
resource "google_logging_project_bucket_config" "bucket" {
  project        = google_project_service.enable_destination_api.project
  location       = var.location
  retention_days = var.retention_days
  bucket_id      = var.name

# Workaround : waiting for fiw https://github.com/terraform-google-modules/terraform-google-log-export/issues/170
# log-export is copied to locals modules app/modules (with lifecycle to ignore locked attribut)

    lifecycle {
    ignore_changes = [
      locked
    ]
  }
}
